### ISSUE TITLE
The capsule in the last level is difficult to see

### ISSUE THREAD

**Issue by lggauci** [Adamcaruana/IMTAssignment2016#12]

**Comment by Adamcaruana** [comment]
I agree with your issue. I have changed the capsule from black to light gray. 
![image](https://cloud.githubusercontent.com/assets/17250422/15288639/76c8c400-1b6c-11e6-80e8-61916737ae69.png)
